constructor/fcp.py

```diff
@@ -8,17 +8,19 @@
 """
 from __future__ import absolute_import, division, print_function
 
-from collections import defaultdict
+from collections import defaultdict, Counter
 import json
 import os
 from os.path import isdir, isfile, join, splitext
+from itertools import groupby, chain
```

Comment:
`chain` is also unused

```suggestion
from itertools import groupby
```

Reply:
Thanks. Not used to not having linter!